lib/sandbox.js

```diff
@@ -66,9 +66,12 @@ Sandbox.prototype.create = function (codeOrUrl, options, cb) {
     }
 
     var self = this;
-    var promise = this.createToken(options)
-        .then(function (token) {
-            return new Webtask(self, token, { meta: options.meta });
+    var token_options = defaults({}, options, { include_webtask_url: true });
+    var promise = this.createToken(token_options)
+        .then(function (result) {
+            return token_options.include_webtask_url
+                ? new Webtask(self, result.token, { meta: options.meta, webtask_url: result.webtask_url })
+                : new Webtask(self, result, { meta: options.meta });
```

Comment:
I now see the added logic below. This dual return isn't something I'm a huge fan of but I recognize that there are few alternatives.